src/character_display.cpp

```diff
@@ -642,6 +646,57 @@ struct HeaderSkill {
     }
 };
 
+int display_empty_handed_base_damage( const Character &you )
```

Comment:
```suggestion
int character_display::display_empty_handed_base_damage( const Character &you )
```
i think this isn't namedspaced (the declaration is namespaced in headers)